packages/progress-indicator/src/LionProgressIndicator.js

```diff
@@ -0,0 +1,110 @@
+import { svg, css, LitElement } from '@lion/core';
+import { localize, LocalizeMixin } from '@lion/localize';
+
+export class LionProgressIndicator extends LocalizeMixin(LitElement) {
+  static get styles() {
```

Comment:
By adding this, these styles will be added to any sub-implementation that uses this to extend the base styles: 

```
static get styles() {
   return [...super.styles, css`...`]
}

So if you extend this component specifically you need to NOT include the styles from super. Is that a pattern that you use elsewhere?

Reply:
In all cases so far we have some minimal styling we would like to reuse. In e.g. the button we have put commends like `/* if you extend, please overwrite */` inside the css on parts that can be overwritten.

Reply:
I think the button is not the right component to take as an example.

Ideally, it should always be:

```
static get styles() {
   return [...super.styles, css`...`]
}
```

I think because of all reasons mentioned (keyframe name collision, lion prefixed classes, redundant css) it 
would be way nicer to give this component empty styles by default (maybe only keep inline-block) and create an extension demo like here: https://github.com/ing-bank/lion/blob/master/packages/collapsible/README.md


The main section of the demo can then just explain what it does: a11y and i18n.
Unless @daKmoR sees harm in this approach.

Reply:
it's fine for now... we will need to improve the lion demos at some point... but that's for later

Reply:
I implemented the approach described by @tlouisse